content/docs/sidebar.yaml

```diff
@@ -1070,6 +1071,20 @@
       slug: introduction/roadmap
       icon: roadmap
       tag: updated
+- section: Workflows
+  items:
+    - title: Neon Twin
+      slug: guides/neon-twin-intro
+      icon: partner-guide # TODO change icon
```

Comment:
we have an icon ready:

icon: workflows

Reply:
Hey @danieltprice. I've had PP create a new icon for Neon Twin. I've updated the sidebar.yaml and added the new svg.